src/stores/Staking.ts

```diff
@@ -0,0 +1,43 @@
+import { createStore } from 'pinia';
+
+export type StakingState = {
+    activeValidator: ValidatorData | null,
+}
+
+export type ValidatorData = {
+    name: string | null,
+    icon: string | null,
+    label: string | null,
+    trust: number | .0,
+    payout: Array<number> | [0, 0, 0, 0, 0],
```

Comment:
` | [0, 0, 0, 0, 0]` is superfluous.
Generally more precise typing would be
```suggestion
    payout: [number, number, number, number, number],
```